.asf.yaml

```diff
@@ -0,0 +1,10 @@
+github:
+  description: "Upserts Deletes And Incremental Processing on Big Data."
+  homepage: https://hudi.apache.org/
+  labels:
+    - hudi
+    - apachehudi
+    - datalake
+    - delta
+    - incremental processing
+    - bigdata
```

Comment:
let's add `stream processing`, `data integration` (we support deltastreamer), `apachespark`?